CyrillicFilters/RussianFilter/sections/specific.txt

```diff
@@ -727,7 +727,7 @@ zr.ru#$?#div[class^="styled__SuspendableElement-"]:has(> div[id^="unit_"]) { rem
 senkuro.com##div[id^="R-A-"]
 afisha.ru#?#div[data-test="PAGE-BLOCK"] > div > section:has(> div[data-test="SECTION-CONTENT"] > div > div:contains(Реклама))
 doramiru.org##.speedbar + div:has(> div[id^="adfox_"])
-lostfilm.tv##.lstfml > a[style^="display:block;width:100%;padding-top:"]
+lostfilm.today,lostfilm.tv##.lstfml > a[style^="display:block;width:100%;padding-top:"]
```

Comment:
I only see its logo there. It's best to enable AdGuard Other Annoyances.

Reply:
ok but big placeholder hence added to existing category

Reply:
What's the big placeholder?